app/src/main/java/com/ataulm/whatsnext/search/SearchActivity.kt

```diff
@@ -19,11 +25,13 @@ class SearchActivity : BaseActivity() {
     private lateinit var bottomSheetBehavior: BottomSheetBehavior<View>
 
     private val navigator = navigator()
-    private val filmSummariesAdapter = FilmSummariesAdapter(object : FilmSummaryViewHolder.Callback {
+    private val filmSummaryCallback = object : FilmSummaryViewHolder.Callback {
         override fun onClick(filmSummary: FilmSummary) {
             viewModel.onClick(filmSummary)
         }
-    })
+    }
+    private val popularFilmsAdapter = PopularFilmsThisWeekAdapter(filmSummaryCallback)
+    private val filmSummariesAdapter = FilmSummariesAdapter(filmSummaryCallback)
```

Comment:
search results and the initial list of movies both use the same view/holder/callback